blessed/terminal.py

```diff
@@ -1100,20 +1100,36 @@ def strip_seqs(self, text):
         """
         return Sequence(text, self).strip_seqs()
 
-    def split_seqs(self, text, **kwds):
+    def split_seqs(self, text, maxsplit=0, flags=0):
```

Comment:
I don't think flags can work here, so we may want to drop it. I get this error:
`ValueError: cannot process flags argument with a compiled pattern`